src/modules/colorPicker/ColorPickerUI/App.xaml.cs

```diff
@@ -27,23 +27,27 @@ protected override void OnStartup(StartupEventArgs e)
             _args = e?.Args;
 
             // allow only one instance of color picker
-            _instanceMutex = new Mutex(true, @"Global\ColorPicker", out bool createdNew);
+            _instanceMutex = new Mutex(true, @"Local\ColorPicker", out bool createdNew);
             if (!createdNew)
             {
                 _instanceMutex = null;
-                Application.Current.Shutdown();
+                Environment.Exit(0);
                 return;
             }
```

Comment:
Is it possible that a new instance will not be created because the old one has not existed yet? For example, when restarting PowerToys as administrator. It seems as we do not need to handle it when running from PowerToys. Anyway, it should be addressed in a separate issue.

Reply:
It's possible in that scenario, but unlikely, and even if that happens, you can still use ColorPicker normally because pressing the shortcut key will launch it again.